aws/terraform/aws/autoscaler.tf

```
@@ -0,0 +1,100 @@
+terraform {
+  required_version = ">= 0.12.6"
+}
+
+# Create IAM role + automatically make it available to cluster autoscaler service account
+module "iam_assumable_role_admin" {
+  source                        = "terraform-aws-modules/iam/aws//modules/iam-assumable-role-with-oidc"
+  version                       = "~> v2.6.0"
```

I think this should be changed to:

```suggestion
  version                       = ">= v2.6.0"
```

Else I get both issues with the terraform version (needs to downgrade), or version conflicts with the aws provider:

```console
No provider "aws" plugins meet the constraint ">= 2.28.1,>= 2.55.0,>= 3.10,~> 2.23".

The version constraint is derived from the "version" argument within the
provider "aws" block in configuration. Child modules may also apply
provider version constraints. To view the provider versions requested by each
module in the current configuration, run "terraform providers".

To proceed, the version constraints for this provider must be relaxed by
either adjusting or removing the "version" argument in the provider blocks
throughout the configuration.


Error: no suitable version is available
```

---

All versions have been updated (see other comment).